tests/dummy/app/templates/nodes/index.hbs

```diff
@@ -4,7 +4,7 @@
     {{input type="text" value=title placeholder="Node's title" required=true}}
     <label>Description (optional):</label>
     {{input type="text" value=description placeholder="Node's description"}}
-    <p><button {{action "createNode" title description}}>Create Node</button></p>
+    <p><button onclick={{action 'createNode' title description}}>Create Node</button></p>
```

Comment:
Shouldn't this work without specifying `onclick`?

I've been using that syntax on some of my recent PRs, eg
https://github.com/CenterForOpenScience/ember-osf/pull/61/files#diff-0c1696451128e99e21bc7e35c6983fadR6